src/compas/datastructures/mesh/_mesh.py

```diff
@@ -2617,7 +2618,7 @@ def vertex_curvature(self, vkey):
 
         """
 
-        return 2 * pi - sum([angle_points(mesh.vertex_coordinates(vkey), mesh.vertex_coordinates(u), mesh.vertex_coordinates(v)) for u, v in pairwise(self.vertex_neighbors(vkey, ordered = True) + self.vertex_neighbors(vkey, ordered = True)[:1])])
+        return 2 * pi - sum([angle_points(self.vertex_coordinates(vkey), self.vertex_coordinates(u), self.vertex_coordinates(v)) for u, v in pairwise(self.vertex_neighbors(vkey, ordered = True) + self.vertex_neighbors(vkey, ordered = True)[:1])])
 
```

Comment:
we should split up this kind of unreadableness. i think i showed one too many times how great one-liners are and now some of our guys write everything they can that way :)

Reply:
lol, this was an impressive one-liner!